.appveyor.yml

```diff
@@ -0,0 +1,53 @@
+environment:
+  ARMADILLO_DOWNLOAD: "http://ftp.fau.de/macports/distfiles/armadillo/armadillo-8.400.0.tar.xz"
```

Comment:
Since this appveyor config is a new addition, might as well use the latest available version of Armadillo (9.800.3).
Armadillo 8.400 is unsupported and pretty ancient at this point; we're not forced to use that version for this config.
We're only forced to use 8.400 on Ubuntu, due to Ubuntu 18.04 LTS.

(Ubuntu 18.04 will soon be a distant memory anyway, as the new 20.04 LTS release is just around the corner. I suspect many Ubuntu users will quickly update to it.)

Reply:
I see what you're saying, but actually I think it's important to build against 8.400.0 here, as it will help ensure that someone committing code isn't committing code that would break on our oldest allowed Armadillo version.  When newer Ubuntu (and RHEL) versions become standard we can bump the minimum Armadillo version, but for now I'm actually still hearing from people who are using Ubuntu 16.04...!